CONTRIBUTING.md

```diff
@@ -170,20 +200,36 @@ included in the project:
 8. [Open a Pull Request](https://help.github.com/articles/about-pull-requests/)
     with a clear title and description against the `main` branch.
 
-__IMPORTANT__: By submitting a patch, improvement, or new feature, you agree to allow the maintainers of slurm-charms to
-license your contributions under the terms of the [Apache Software License, version 2.0](./LICENSE).
-
 ## Discussions
 
-GitHub's discussions are a great place to connect with other users of the slurm charms as well as discuss potential
-features and resolve personal support questions. It is expected that the users must remain respectful of
-each other. Discussion moderators reserve the right to suspend discussions and/or delete posts that do not follow
-this rule.
+GitHub Discussions is a great place to connect with other Charmed HPC users to
+discuss potential enhancements, ask questions, and resolve issues. Charmed HPC users
+should remain respectful of each other. Discussion moderators reserve the right to
+suspend discussions and/or delete posts that do not follow this rule.
 
 ## Code guidelines
 
 The following guidelines must be adhered to if you are writing code to be merged into the main code base:
 
+### Monorepo
+
+* We use a mono repository (monorepo) for tracking the development of the Slurm charms.
+  All Slurm-related charms must be contributed to this repository and not
+  broken out into its own standalone repository. Here's why:
+
+  * We can test against the latest commit to the Slurm charms rather than
+    pull what is currently published to edge on Charmhub.
+    * If the integration tests we're to pull the Slurm charms from the edge
+      branch on Charmhub, we can potentially waste CI minutes as we cannot
+      test breaking changes easily compared to just using the latest commit.
+  * It's easier to enable CI testing for development branches. We can test
+    the `experimental` development branch in the CI rather pipeline rather than need to create a separate workflow file off of `main`.
```

Comment:
```suggestion
    the `experimental` development branch in the CI pipeline rather than needing to create a separate workflow file off of `main`.
```